packages/benchmark/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "n8n-benchmark",
```

Comment:
Should we ignore this package when doing a release? In the same way we don't ship test code.

Reply:
Yes we don't need to publish this package at the moment. The only thing we need is the docker image, which is built from `master`. So the benchmark cli has its own release cycle